documentation/views/portfolio.rst

```diff
@@ -32,16 +39,16 @@ A consumer will only have costs, while a prosumer may have both costs and revenu
 A supplier always has both costs and revenues, since it trades energy both with its customers and with external markets.
 Finally, the financial statements show the total profit or loss per asset.
 
-Statements about flexibility actions
+Statements about flexible schedules
 ----------------------------------
 
-The bottom table lists the effects of flexibility actions for each asset type in the user's portfolio.
-Separate columns are stated for each type of action, e.g. curtailment and shifting (see :ref:`action_types`), with relevant total volumes within the selected time window.
+The bottom table lists the effects of flexible schedules for each asset type in the user's portfolio.
+Separate columns are stated for each type of scheduled deviation from the status quo, e.g. curtailment and shifting (see :ref:`opportunity_types`), with relevant total volumes within the selected time window.
 [#f1]_
 
-Costs and revenues are calculated based on the internal method for profit sharing.
-Asset owners that provide flexibility actions via the platform will generate revenues.
-Suppliers that order flexibility action via the platform will generate both costs and revenues, where the revenues come from interacting with external markets.
+Costs and revenues are calculated based on an internal method for profit sharing (TODO: explain).
```

Comment:
This assumes a contractual relationship between multiple stakeholders, for example:

- between Aggregator and Prosumer, where the Aggregator sells the balancing power to a third party and shares the profits with the Prosumer according to some contracted method for profit sharing. In this case the stated costs and revenues for the Prosumer may be after deducting the Aggregator fee (which typically include price components per flex activation and price components per unit of time, but may include arbitrarily complex price components).
- between ESCo and Prosumer, where the ESCo advises the Prosumer to optimise against e.g. dynamic prices. Likewise, stated numbers may be after deducting the ESCo fee.

Alternatively, we can assume that all profit from activating flexibility goes to the Prosumer, or simply report the profits before sharing (and before deducting any service fees).

Reply:
I will use this content in my work on #45, thanks.